packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/billingAccount.routing.js

```diff
@@ -99,6 +99,10 @@ export default /* @ngInject */ ($stateProvider) => {
         $state.href('telecom.telephony.billingAccount.guides', {
           billingAccount: billingAccountId,
         }),
+      billingDepositLink: /**/ ($state, billingAccountId) =>
```

Comment:
```suggestion
      billingDepositLink: /* @ngInject */ ($state, billingAccountId) =>
```

Reply:
Thx for that. I really need to sleep! :-P